pylint/checkers/utils.py

```diff
@@ -38,6 +38,15 @@
     nodes.DictComp,
     nodes.GeneratorExp,
 )
+WITH_IGETATTR = (
```

Comment:
Isn't it possible to use ``has_attr("igetattr"`` and have something generic instead ?

Reply:
Sure, I just figured this was more explicit.

Reply:
> this was more explicit.

Sure but if another class with ``igetattr`` appear we'd need to remember to add it there, which is unlikely to be caught in review, right ?

Reply:
That makes sense, will change.